http/vibe/http/client.d

```diff
@@ -182,6 +150,29 @@ void requestHTTP(URL url, scope void delegate(scope HTTPClientRequest req) reque
 	assert(!cli.m_responding, "HTTP client still responding after return!?");
 }
 
+private bool isTlsNeed(in URL url, in HTTPClientSettings settings)
+{
+	version(UnixSocket) {
+		enforce(url.schema == "http" || url.schema == "https" || url.schema == "http+unix" || url.schema == "https+unix", "URL schema must be http(s) or http(s)+unix.");
+	} else {
+		enforce(url.schema == "http" || url.schema == "https", "URL schema must be http(s).");
+	}
+	enforce(url.host.length > 0, "URL must contain a host name.");
+	bool use_tls;
```

Comment:
Instead of creating a variable, you can just `return` when it is set, because the variable is only ever assigned to once

Reply:
I want to avoid changes as much as possible so as not to add any error here. It's okay if such a variable stays here for a while. And personally I do not like many exit points.